buildkite/bazelci.py

```diff
@@ -580,25 +590,108 @@ def _get_buildkite_token(self):
             else "buildkite-untrusted-api-token",
         )
 
-    def _open_url(self, url):
+
+    def _open_url(self, url, params = None):
         try:
+            params_str = ""
+            if params:
+                for k, v in params:
+                    params_str = params_str + "&{}={}".format(k, v)
```

Comment:
string concatenation in a loop is not nice. Better:
params_str = "".join("&{}={}".format(k, v) for k, v in params)

Reply:
Thank you!